GUIDE.md

```diff
@@ -156,6 +156,33 @@ See the Nix notes in the appendix at the end of this guide.
 As of this writing, they reflect using Nix instead of Stack, so updates and
 reconsideration of this process is called for.
 
+### Arch Linux
+
+To install dependencies, run this command as `root`:
+
+    pacman -S git ghc cabal-install alex happy haddock postgresql
```

Comment:
we don't install cabal-install, alex, happy, or haddock system-wide anywhere else anymore. We just install those with stack. Doesn't that make more sense?